docs/docs/install/kubernetes.md

```diff
@@ -47,17 +49,35 @@ tracee-fcjmp   1/1     Running   0          4m11s
 ```
 </details>
 
-## Interacting with Tracee
+Once installed, Tracee immediately starts producing events. Since Tracee is deployed as a DaemonSet, a Tracee Pod is running on every node in the cluster. Every Tracee Pod is monitoring the node it is running on.
 
-Once installed, Tracee immediately starts producing system activity events, such as processes and containers activity, network activity, and more. To see the events that Tracee produces, use can use the `kubectl logs` command.
+## Viewing Events
+
+The easiest way to tap into the log stream of all Tracee Pods is with the `kubectl logs` command:
 
 ```console
-kubectl logs -f daemonset/tracee -n tracee-system
```

Comment:
the Helm Chart is usually installed in the tracee-system namespace, why don't we add it here?